src/Sylius/Bundle/CoreBundle/Resources/config/services.xml

```diff
@@ -239,9 +239,6 @@
         <service id="sylius.order_item_names_setter" class="Sylius\Component\Core\Order\OrderItemNamesSetter" />
         <service id="Sylius\Component\Core\Order\OrderItemNamesSetterInterface" alias="sylius.order_item_names_setter" />
 
-        <service id="sylius.grid_filter.entities" class="Sylius\Component\Core\Grid\Filter\EntitiesFilter">
```

Comment:
@GSadee Should I also mention that removed service?

Reply:
Yes, add a note to the UPGRADE file, and ideally also a deprecation to 1.14 🙊